sw/device/lib/dif/dif_padctrl.c

```diff
@@ -0,0 +1,180 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#include "sw/device/lib/dif/dif_padctrl.h"
+
+#include "padctrl_regs.h"
+#include "sw/device/lib/base/mmio.h"
+
+#define PADCTRL_REGISTER_WIDTH_IN_BYTES (PADCTRL_PARAM_REG_WIDTH / 8)
```

Comment:
In general, if a `#define` is not needed in a constant-evaluation location (i.e. in an array bound), prefer a `static const` instead.

Reply:
Done